include/Chess.h

```diff
@@ -0,0 +1,40 @@
+#pragma once

```

Comment:
The use of #pragma once at the beginning of the header file is good practice and helps prevent multiple inclusions of the file. However, it is not standard C++ and is not guaranteed to be supported by all compilers. A more portable alternative is to use include guards:
#ifndef BISHOP_H
#define BISHOP_H